concent_api/api_testing_common.py

```diff
@@ -72,6 +74,11 @@ def assert_condition(actual, expected, error_message = None):
         raise TestAssertionException(message)
 
 
+def assert_iterables_content_is_equal(actual: Iterable[str], expected: Iterable[str]) -> None:
+    if not sorted(actual) == sorted(expected):
+        raise TestAssertionException(f'actual and expected lists are not equal. Actual {actual}. Expected {expected}')
```

Comment:
Please reformat the message:
`f' Content of iterables is not equal: expected={expected}, actual={actual} '`

Reply:
done